index.html

```diff
@@ -11,23 +11,16 @@
         <h2 class="title is-4">
            Get/Deliver Food (still happening!)
         </h2>
-
+     
         <p>
-            We’re excited to have two delivery partnerships available in the Denver metro area: one with Joy’s Kitchen in Lakewood,
-            and one with Growing Home in Westminster.
+            RMMAN is still delivering food to families in and around Denver, to learn more about what we do visit our <a href="https://linktr.ee/rmmutualaid" target="_blank" rel="noopener noreferrer">linktree page.</a>
         </p>
 
-        <p>
-            {% include btn.html href="/volunteer/delivery/" class="is-success is-large is-fullwidth" text="Learn More" %}
```

Comment:
If you want a button for linktree, just replace the `href` and `text` attributes on this Liquid Include.